main/src/library/flix/actor/Actor.flix

```diff
@@ -23,7 +23,7 @@ pub enum ActorResult[s, e] {
     ///
     /// Indicates that the actor was unable to complete it works and aborted.
     ///
-    case Crash(s, e)
+    case Crash(e)
```

Comment:
An actor could returns a closure that represents its own continuation, i.e. a function `Mailbox -> ActorResult[e]` and that we could still allow a form of resumption. Thoughts?

Reply:
Sounds like a good compromise to be able to resume from the previous state without having the state as part of the ActorDef.